docs/UserGuide.md

```diff
@@ -487,7 +487,20 @@ Selects a timeslot to be added to the user timetable.
 Format: `select /module MODULE_CODE /type LESSON_TYPE /code CLASS_NO`
 
 * The `CLASS_NO` will be a number.
-* The `LESSON_TYPE` can be either *lecture*, *tutorial*.
+* The `LESSON_TYPE` can be any of the following:  
+  *  TUTORIAL                   represented by inputs starting with `TUT`
```

Comment:
Don't tell them how we are parsing it lol. Just tell them what is accepted. I suggest just give them two options - spell out in full or short form.